src/plots/cartesian/axes.js

```diff
@@ -507,7 +507,6 @@ axes.autoBin = function(data, ax, nbins, is2d) {
         var distinctData = Lib.distinctVals(data),
             msexp = Math.pow(10, Math.floor(
                 Math.log(distinctData.minDiff) / Math.LN10)),
-            // TODO: there are some date cases where this will fail...
```

Comment:
removed because I have no idea anymore what this was about. We'll wait for a bug report...